src/EmployerAan/SFA.DAS.EmployerAan.Api/Controllers/MemberProfilesController.cs

```diff
@@ -27,8 +29,23 @@ public MemberProfilesController(IMediator mediator)
         bool @public = true)
     {
         var memberProfileWithPreferences = await _mediator.Send(new GetMemberProfileWithPreferencesQuery(memberId, requestedByMemberId, @public), cancellationToken);
-        var employerMemberSummary = await _mediator.Send(new GetEmployerMemberSummaryQuery(memberProfileWithPreferences.AccountId), cancellationToken);
 
-        return Ok(new GetMemberProfileWithPreferencesModel(memberProfileWithPreferences, employerMemberSummary));
+        var isApprenticeshipSectionShared = memberProfileWithPreferences.Preferences.Single(x => x.PreferenceId == Constants.PreferenceIds.Apprenticeship).Value;
+
+        if (@public && !isApprenticeshipSectionShared)
+        {
+            return Ok(new GetMemberProfileWithPreferencesModel(memberProfileWithPreferences, null, null));
+        }
+
+        if (memberProfileWithPreferences.UserType == MemberUserType.Apprentice)
+        {
+            var myApprenticeship = await _mediator.Send(new GetMyApprenticeshipQuery(memberProfileWithPreferences.ApprenticeId), cancellationToken);
+            return Ok(new GetMemberProfileWithPreferencesModel(memberProfileWithPreferences, myApprenticeship, null));
+        }
+        else
```

Comment:
dont need an else here, it will just happen.

Reply:
Done